site/content/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+headless: true
+addSocialPreview: true
+description: Welcome to ZAP!
+title: The ZAP Homepage
```

Comment:
No problem changing the description/title...I just put this here so we'd have something, but feel free to edit as you see fit!

Reply:
Looks liek a good starting point, we can easily change this in the future if we decide to :)